boa/src/builtins/object/mod.rs

```diff
@@ -84,24 +85,28 @@ impl Object {
     ///
     /// [spec]: https://tc39.es/ecma262/#sec-object.create
     /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
-    pub fn create(_: &Value, args: &[Value], interpreter: &mut Context) -> Result<Value> {
+    pub fn create(this: &Value, args: &[Value], ctx: &mut Context) -> Result<Value> {
```

Comment:
I'm a bit confused why `create` accepts a `this` argument. Isn't it a static method?

Reply:
I think it's because it needs to invoke `Object::define_properties`, which will need a `this` because it is also a BuiltIn function.

Reply:
But looking at line 158 it doesn’t look like it does anything with the first value

Reply:
Would you preffer to not have `this` in `create` and invoke `define_properties` with `&Value::Undefined`, or some other non-relevant value?